test/Mysql/StmtTest.php

```diff
@@ -0,0 +1,77 @@
+<?php
+/*
+ * This file is part of the OpCart software.
```

Comment:
No, it's not. :stuck_out_tongue_winking_eye:

Reply:
ops... 😄  This should not be here